src/app/[locale]/incident/add/components/MapDialog.tsx

```diff
@@ -115,20 +160,36 @@ const MapDialog = ({
 
   // Handle click on map, setIsMapSelected to true
   // TODO: Reset selectedFeatures if click was right on map? (open for discussion)
-  const handleMapClick = (event: MapLayerMouseEvent) => {
+  const handleMapClick = async (event: MapLayerMouseEvent) => {
     updatePosition(event.lngLat.lat, event.lngLat.lng)
-
     setIsMapSelected(true)
```

Comment:
Ik vind de naam isMapSelected denk ik een beetje verwarrend. Bij zowel marken als feature geselecteerd heb je iets op de kaart geselecteerd toch? Zullen we hiervan maken featureSelected oid?